src/test/java/org/eclipse/yasson/internal/AnnotationIntrospectorTestFixtures.java

```diff
@@ -31,7 +31,7 @@
         return AccessController.doPrivileged((PrivilegedAction<Constructor<?>[]>) clazz::getDeclaredConstructors);
     }
 
-    public static class ObjectWithoutAnnotatedConstructor implements ProvidesParameterRepresentation {
```

Comment:
I didn't name these test objects originally, but I do find the original class names to be more clear. Can we please revert the names back?

Reply:
I reverted them back. I named them originally and thought, they may get to long. But it is much cleaner for this branch to let it as it was.